tc/sgx/trusted_worker_manager/enclave_untrusted/enclave_bridge_wrapper/signup_info.h

```diff
@@ -24,48 +24,31 @@
 
 class SignupInfo {
 public:
-    friend SignupInfo* deserialize_signup_info(const std::string& s);
+    SignupInfo() {}
 
     std::string serialize() const {
         return serialized_;
     }
 
+    static tcf_err_t DeserializePublicEnclaveData(
+        const std::string& public_enclave_data,
+        std::string& verifying_key,
+        std::string& encryption_key,
+        std::string& encryption_key_signature);
```

Comment:
Do we need to keep this as static member function ? Or can it be made a non-static member function which can access the member variables verifying_key, encryption_key and encryption_key_signature ?

Reply:
The function was static earlier, i moved it under the class as is. I will make the function non-static and access the variables through instance.